diesel/src/sql_types/mod.rs

```diff
@@ -332,14 +332,12 @@ pub struct Time;
 /// - [`std::time::SystemTime`][SystemTime] (PG only)
 /// - [`chrono::NaiveDateTime`][NaiveDateTime] with `feature = "chrono"`
 /// - [`time::PrimitiveDateTime`] with `feature = "time"`
-/// - [`time::OffsetDateTime`] with `feature = "time"`
```

Comment:
There are such impls for the mysql backend. Probably we should change it to (mysql only) instead of removing these from the list